lib/views/login_register/forgot_password.dart

```diff
@@ -43,20 +63,49 @@ class ForgotPasswordScreen extends StatelessWidget {
                       ),
                       color: Colors.white,
                       borderRadius: BorderRadius.circular(12)),
-                  child: const TextField(
-                    decoration: InputDecoration(
+                  child: TextField(
+                    autocorrect: false,
+                    enableSuggestions: false,
+                    controller: _email,
+                    decoration: const InputDecoration(
                       hintText: 'Email',
                       hintStyle: TextStyle(color: Colors.grey),
                       border: InputBorder.none,
                       labelStyle: TextStyle(color: Colors.black),
                     ),
-                    style: TextStyle(color: Colors.black),
+                    style: const TextStyle(color: Colors.black),
                   ),
                 ),
                 Padding(
                   padding: const EdgeInsets.all(30),
                   child: BigButton(
-                    onPressed: () => navigator.push(const PasswordResetSentScreen()),
+                    onPressed: () async {
+                      final email = _email.text;
+                      try {
+                        await FirebaseAuth.instance.sendPasswordResetEmail(email: email);
```

Comment:
Currently all FirebaseAuth calls that involve a url link do not redirect back to the app. From my research it is common to use dynamic links in firebase for these redirects to the app. But google is going to deprecate this next year so it is not recommended. Are we okay with leaving it the way it is for now or should we look into alternatives to redirect to the app after clicking email link?